content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning.md

```diff
@@ -145,32 +147,34 @@ the version.
 
 ## Webhook conversion
 
-The above example has a None converter between versions which only set `apiVersion` field
-on conversion and do not change the rest of the object. API server supports webhook conversion that
+{{< note >}}
+Webhook conversion is intorduced in kubernetes 1.13 as an alpha feature. To use this feature, the
+`CustomResourceWebhookConversion` feature get should be set. Please refer to [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) documentation for more information.
```

Comment:
introduced

Reply:
I looked at the preview, the Note text (Note**) is not rendered probably because the dev-1.13 branch does not include the updated sass from the master?

Reply:
I used grep to compare apiserver vs API server which may not be a good indicator. I will go with API server

Reply:
> if you referring to the "addressed comments ..." commit msg, that was temporary and will go away.